media/js/webAudioApiForDesigners.js

```diff
@@ -0,0 +1,219 @@
+// Browser detection required (guess which browsers are at faul =P)
```

Comment:
Is this something that you wrote, or did you copy from somewhere else? we need to make sure we update our licensing information with details about this file